### ISSUE TITLE
Recognize custom test report locations in build.gradle

### ISSUE THREAD

**Issue by kathrynkodama** [OpenLiberty/liberty-dev-vscode-ext#23]
A test report location can be specified in a `build.gradle` the following way:
`test.reports.html.destination = file("$buildDir/reports/intTests”)`

The extension should recognize custom locations and open the corresponding html file as opposed to simply looking in the default location for the test report (`/build/reports/tests/test/index.html`).

**Comment by ericglau** [comment]
One way is to scan the console output for the report location, but this is only printed when there are failures. e.g.
```
FAILURE: Build failed with an exception.
* What went wrong:
Execution failed for task ':test'.
> There were failing tests. See the report at: file:///Users/eric/git/demo-devmode/build/reports/intTests/index.html
```

**Comment by kathrynkodama** [comment]
Test reports can also be specified beneath a test tag, ensure these cases are covered:
```
  test {
        reports.html.destination = file("$buildDir/reports/unit")
        reports.junitXml.destination = file("$buildDir/test-results/unit")
        exclude '**/it/**'
    }
```

**Comment by kathrynkodama** [comment]
Fixed with #48
